packages/vuetify/src/labs/VDataTable/__tests__/VDataTable.spec.ts

```diff
@@ -937,7 +937,7 @@ describe.skip('VDataTable.ts', () => {
         headers: testHeaders,
         items: testItems,
         itemsPerPage: 5,
-        itemClass: (item: Object) => ({
+        itemClass: ({ item }: Object) => ({
           'first-class': item.fat < 10,
           'second-class': item.protein > 4.0,
         }),
```

Comment:
Should this test be updated with new markup? `itemClass` was previously supporting `Record<string, boolean>` but the new function signature expects `string[]`. Should this be changed to the V2 signature, so migration is easier for this prop?